packages/destination-actions/src/destinations/braze/createAlias2/__tests__/snapshot.test.ts

```
@@ -0,0 +1,93 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../../lib/test-data'
+import destination from '../../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const actionSlug = 'createAlias2'
+const destinationSlug = 'Braze'
+const seedName = `${destinationSlug}#${actionSlug}`
+
+describe(`Testing snapshot for ${destinationSlug}'s ${actionSlug} destination V2 action:`, () => {
+  it('fails if sync mode is not add', async () => {
+    const action = destination.actions[actionSlug]
+    const [eventData, settingsData] = generateTestData(seedName, destination, action, false)
+
+    const event = createTestEvent({
+      properties: eventData
+    })
+
+    await expect(
+      testDestination.testAction(actionSlug, {
+        event: event,
+        mapping: { ...event.properties, __segment_internal_sync_mode: 'upsert' },
+        settings: settingsData,
+        auth: undefined
+      })
+    ).rejects.toThrowError()
```

NIT: Make sure you are not swallowing other errors as this is a catch all for all the exceptions. It's better if you compare against the actual exception you are throwing.